src/containers/AllStreamCard/index.tsx

```diff
@@ -1,54 +1,61 @@
-import React from 'react';
+'use client';
+
+import React, { useState } from 'react';
 import Image from 'next/image';
 import { useRouter } from 'next/navigation';
 
 import CCard from 'src/components/CCard';
 import { Pages } from 'src/constants/pages';
 import CButton from 'src/components/CButton';
 
-import cardLogo from 'public/images/card.svg';
+import close from 'public/images/close.svg';
```

Comment:
rename to closeLogo

Reply:
it's nor close logo, its the logo you put in AllStreamCard which is an illustration.